modules/asset_import/assimp.h

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.
```

Comment:
Please use the official copyright header from other .h/.cpp files.

Reply:
And you're 2 years off ;)

Reply:
It only took two years ;)